File: Objective-C Proj./Onlineshop_Aarti_IPhone_Project/Myproject/Myproject/homeViewController.m
//
//  homeViewController.m
//  Myproject
//
//  Created by tops on 12/28/14.
//  Copyright (c) 2014 disha. All rights reserved.
//

#import "homeViewController.h"

@interface homeViewController ()

@end

@implementation homeViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [NSTimer scheduledTimerWithTimeInterval:1 target:self selector:@selector(goToNext) userInfo:nil repeats:YES];
    // Do any additional setup after loading the view.
}
int cout=0;

//-(void)viewWillAppear:(BOOL)animated
//{
  //  self.navigationController.navigationBar.hidden=YES;
//}

-(void)goToNext
{
    self.view.alpha-=0.1;
    if(cout==4)
    {
        UIViewController *lV=[self.storyboard instantiateViewControllerWithIdentifier:@"registration"];
        [self.navigationController pushViewController:lV animated:YES];
    }
    cout++;
}
- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: SWIFT Proj./device controller 2 3/device controller/Home ViewController.swift
//
//  Home ViewController.swift
//  device controller
//
//  Created by TOPS on 21/03/18.
//  Copyright © 2018 TOPS. All rights reserved.
//

import UIKit
import Firebase
import FirebaseAuth


class Home_ViewController: UIViewController {

    var str=""
    var name=""
    var tm=Timer()

    @IBOutlet var home_nav: UINavigationBar!
    
    @IBOutlet var home_img: UIImageView!
    
    @IBOutlet var welcome_lbl: UILabel!
    
    @IBOutlet var name_lbl: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
     
        name_lbl.text=name
        
        tm=Timer.scheduledTimer(timeInterval: 3, target: self, selector: #selector(Home_ViewController.main), userInfo: nil, repeats: false)
            }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    func main()
    {
        let nxt=storyboard?.instantiateViewController(withIdentifier: "addlist") as! ListViewController
        
        navigationController?.pushViewController(nxt, animated: true)
    }
    
  }
